python/test_dcr_template.py

```diff
@@ -236,18 +233,29 @@ def setUp(self):
         band_name = 'g'
         n_step = 3
         pixel_scale = 0.25
-        self.kernel_size = 5
+        kernel_size = 5
         self.size = 20
         # NOTE that this array is randomly generated with a new seed for each instance.
```

Comment:
Likewise at line 400-ish, right before the loop that generates images in `DcrModelGenerationTestCase.setUp`